### PR TITLE
testing with bats on travisCI

### PR DESCRIPTION
**Pull request by joshuacox**
Refactoring the tests to eliminate some of the explicit echos as this is all implicit on exit status, and swapping the addus url to the transmute url.

### CODE REVIEW

File: .ci/testpub.sh [hunk 1]
```diff
@@ -0,0 +1,12 @@
+#!/bin/sh
+pub_to_test=$1
+
+TMP=$(mktemp --suffix=testpub -d)
+cleanup_tmp () {
+  rm -Rf $TMP
+}
+trap cleanup_tmp EXIT
```
Comment (eolszewski) [inline .ci/testpub.sh:8]: Did not know about trap statements, this is a really excellent practice to make sure you cover your tracks.

File: pubs [hunk 2]
```diff
@@ -1,3 +1,5 @@
...
\ No newline at end of file
+# Or13
+ssh-rsa AAAAB3NzaC1yc2EAAAADAQABAAACAQDAgzNBMM7hJKeJPhcP2Vp2gEUjzU8L3lewyycOHaaxyAcT62J3J1/UGTKpFDMsvGi/+62KQz9DH4BgzC4vnRn/Xrss234vKLo8QvdpycUexN0m1GcyUjrl3IueZzGAZpMMLLb92yRqSyxHEMPvuddRcVtBkOL7aLEx+mk3c2NqldONlDvS0KjI6kswhKDhFE+0YQhqz/lLTOWH+AryIVGe9UYLRxMen7guhTXgYtisXIqI+AZbHfL3CEvYO9nQpXla0EUxdi20R3iS5yZy1cUtJ64rK1i10K01a2dqlyTId5jygwzL2rWmcdrPNDXTXGCcSGg77gUMYaiDJLvyb+pfPumlaY8v2hUNFfFoiRCqqhbwRGvgWB4F438W4eT6ZhLnQ3qVcQ8I7HKM+C7YH3x4uTg9X0H5ZQZgrkSQxcFfFv1l+rKvdLUQ8iWVFyxU/1uCFW83nHcoYLa6pjaoxzmzPeFsm7PxoFiIQQz0IVjDcyfsVte4T8GqkT5nJ0c7QdOxzQTK1GCjoIfc2onSS+8HKnExxaJ4tpLcu+ez4h7wtjhofe3Gjemi6JQ//OTnZ15rOkugzjASgjVoe2tGKRTZMO2/WDRKzGJIZ9OwK1ycn/DIY/cSyJk7+1UqcD2dgZsCDf6m1f6YbzCdduKOyeJqT/pAkIBvEU1HRwO3I52N2w== orie@transmute.industries
+# Josh
+ecdsa-sha2-nistp256 AAAAE2VjZHNhLXNoYTItbmlzdHAyNTYAAAAIbmlzdHAyNTYAAABBBBI0wpMByzpFXTHLV98y4lFm8qB/XrLP8417Db0P0OuCgpQ/+RmaATZ9NjlDgs5jGX8UWe9zEJFq77zmlQHqIrA= josh@flux.transmute.network
+ecdsa-sha2-nistp256 AAAAE2VjZHNhLXNoYTItbmlzdHAyNTYAAAAIbmlzdHAyNTYAAABBBNv6ctZFflkpPgbjkvYne46Yyd/jgjm/aEPkqc5ju01dooa7G2DzPpp9rw5he3mjqgEULedxJW2az8/try0ealU= josh@stealth.gs70.msi.oltorf.transmute.network
```
Comment (eolszewski) [inline pubs:5]: Can you re-add my pubkey?
https://github.com/transmute-industries/pubs/pull/9/commits/7e6d35a09a16589147a877f8436adddf8bbcf60a

**Review by eolszewski — DISMISSED** [review]

File: pubs (hunk 2 above)
Comment (joshuacox) [reply pubs:5]: This recent merge should've fixed that, I should have pulled apparently last night before squashing and merging.

File: .ci/testpub.sh (hunk 1 above)
Comment (joshuacox) [reply .ci/testpub.sh:8]: I'm fairly certain every programming language has some sort of signal handler, for example 
[node](https://nodejs.org/api/process.html).  Mitchell Hashimoto has said this one of the things he had screwed up initially with vagrant, by adding signal handling it became a much more robust application.

**Review by eolszewski — APPROVED** [review]

**Comment by joshuacox** [comment]
Ok this is borken because I set the email address to josh@transmute.industries on stealth's gpg key.  I will fix when I get back there later today.  For now, closing here, and I'll hav another squashed and signed with the gpg key from flux.

**Pull request closed without merging**